providers/base/units/tracers/resource.pxu

```diff
@@ -0,0 +1,12 @@
+id: kernel_config_file
+plugin: resource
+_summary:
+ Kernel config file resource
+command:
+ if [[ -f "/boot/config-$(uname -r)" ]]
+ then
+     echo "detected: true"
+ else
+     echo "detected: false"
```

Comment:
Without `exit 1` here the test will always pass.

Reply:
Oh, this shouldn't exit with 1. Sorry, one copy pasta too many.

Reply:
I though about it maybe you wanted to talk about the tracefs test which I patched too